app-name/src/Create.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+import { data } from './data';
+
+export default function Create({ viewSwap }) {
+  const [imageUrl, setImageUrl] = useState('');
+  const [title, setTitle] = useState('');
+  const [notes, setNotes] = useState('');
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    const inputValues = {
+      title: title,
+      imageUrl: imageUrl,
+      notes: notes,
+    };
+    inputValues.entryId = data.nextEntryId;
+    data.nextEntryId++;
+    data.entries.unshift(inputValues);
+    viewSwap('entries');
+  }
+
+  return (
+    <div data-view="entry-form" className="entry-form">
+      <form className="form" onSubmit={handleSubmit}>
+        <div className="row">
+          <div className="column-full">
+            <div className="top-header">
+              <h2>New Entry</h2>
+            </div>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="column-full column-half">
+            <img
+              src={imageUrl ? imageUrl : '/images/placeholder-image-square.jpg'}
+              alt="placeholder img"
+            />
+          </div>
+          <div className="column-full column-half">
+            <div className="title-box">
+              <label htmlFor="title-text">Title</label>
```

Comment:
Personally, I prefer to wrap the label around the input instead of using `htmlFor`. You can then use css to lay things out like you'd like. But what you've done certainly works!